lib/dynamo-backup.js

```diff
@@ -71,18 +79,25 @@ DynamoBackup.prototype.backupTable = function (tableName, backupPath, callback)
         tableName,
         function(items) {
             items.forEach(function(item) {
-                stream.append(JSON.stringify(item));
-                stream.append('\n');
+                if (firstRecord) {
+                  firstRecord = false;
+                  stream.append('[\n');
```

Comment:
@adambiggs - hate to do this after making you wait so long - but I think this should be an option with the current behavior as a default or the behavior shouldn't change.

Here is my thinking on why it should be an option and why the default should be the current behavior:
- This is a pretty big change in behavior for people currently using the library
- Having the file be a single JSON object would make a streaming restore harder to perform as you have to load and parse the entire file before you can restore the first item. This was my original motivation for having the current behavior.

Let me know if you disagree here.